modules/eks/privatelink.tf

```diff
@@ -0,0 +1,10 @@
+module "privatelink" {
+  source                   = "./submodules/privatelink"
```

Comment:
We need a toggle. this should be off by default. most consumers of the module will not use this. should be driven by a property in the `eks` var

Reply:
added